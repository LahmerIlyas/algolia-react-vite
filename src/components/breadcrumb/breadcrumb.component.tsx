import Typography from '../typography/typography.component'

export const BreadCrumb: React.FC = () => {
  return (
    <div className="flex flex-row align-center gap-x-1 py-4">
      <Typography as="h6" variant="s6" className="font-normal">Homepage</Typography>
      <Typography as="h6" variant="s6" className="font-normal">/</Typography>
      <Typography as="h6" variant="s6" className="font-normal"> Fruit and vegetables</Typography>
    </div>
  )
}
