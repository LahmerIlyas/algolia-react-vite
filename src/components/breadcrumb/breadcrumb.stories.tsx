import React from 'react'
import type { ComponentMeta, ComponentStory } from '@storybook/react'

import { BreadCrumb } from './breadcrumb.component'

export default {
  title: 'Example/BreadCrumb',
  component: BreadCrumb,
} as ComponentMeta<typeof BreadCrumb >

const Template: ComponentStory<typeof BreadCrumb> = () => (
  <React.Fragment>
    <BreadCrumb/>
  </React.Fragment>
)

export const ExampleStory = Template.bind({})
ExampleStory.args = {
}
