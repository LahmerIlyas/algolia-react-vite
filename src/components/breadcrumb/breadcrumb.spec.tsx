import { describe, expect, it } from 'vitest'
import renderer from 'react-test-renderer'
import { BreadCrumb } from './breadcrumb.component'

describe('Given breadcrumb element', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<BreadCrumb />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
