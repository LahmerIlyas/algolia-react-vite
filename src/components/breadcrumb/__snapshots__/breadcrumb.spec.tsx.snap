// Vitest Snapshot v1

exports[`Given breadcrumb element > should renders correctly 1`] = `
<div
  className="flex flex-row align-center gap-x-1 py-4"
>
  <h6
    as="h6"
    className="text-s6 font-normal"
  >
    Homepage
  </h6>
  <h6
    as="h6"
    className="text-s6 font-normal"
  >
    /
  </h6>
  <h6
    as="h6"
    className="text-s6 font-normal"
  >
     Fruit and vegetables
  </h6>
</div>
`;
